### PR TITLE
grid component

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR has the following
1. Simplified Grid 2
2. Simplified Grid 3

### CODE REVIEW

**Review by ranitasahaa1230 — APPROVED** [review]

File: Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    grid-template-columns: repeat(2, 1fr);
+}
+
+.grid-container-three-col {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+}
+
+.box {
+    border: 1px solid black;
```
Comment (ranitasahaa1230) [inline Grid/grid.css]: You can declare the color variables in your root

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good. Clean and readable code

File: Grid/grid.css (hunk 1 above)
Comment (prakashsakari) [reply Grid/grid.css]: thank you for pointing out this, I missed adding color variable over here

Comment (prakashsakari) [reply Grid/grid.css]: updated, thanks

**Review by AshwiniSetty — APPROVED** [review]

**Pull request merged**